radio/src/opentx.h

```diff
@@ -1012,6 +1014,12 @@ constexpr uint8_t SD_SCREEN_FILE_LENGTH = 64;
 #include "bluetooth.h"
 #endif
 
+//OW
+#if defined(TELEMETRY_MAVLINK)
```

Comment:
You should not add up to `opentx.h`. This is legacy. Please use that include everywhere needed, where it is needed. Adding everything to `opentx.h` dramatically causes a full rebuild when anything included in that file is modified.